Marlin/MarlinSerial.cpp

```diff
@@ -310,3 +315,155 @@ MarlinSerial customizedSerial;
 #if defined(USBCON) && ENABLED(BLUETOOTH)
   HardwareSerial bluetoothSerial;
 #endif
+
+#if ENABLED(EMERGENCY_PARSER)
+
+  #define EMERGENCY_CODES "M112,M108,M410"
```

Comment:
The EMERGENCY_CODES here should be defined in the Configuration_adv.h, not in the MarlinSerial.cpp, otherwise, language.h (for the M115) won't have access to it.